learn/wavemaker-release-notes/v11-7-2.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "WaveMaker 11.7.2 - Release date: 3 June 2024"
+id: "v11-7-2"
+sidebar_label: "v11.7.2 - latest"
+---
+
+This release includes enhancements, such as Artifact Version Management, Notifications Response in Project Hub, some bug fixes.
+
+---
+
+## Enhancements
+
+- **Artifact Version Management**: Optimized artifact version handling to reduce the response size for artifact updates.
+- **Notifications Response in Project Hub**: Notification filtering at the API level to enhance the user experience in Project Hub.
+
+## Bug Fixes
+
+- Fixed issue with nested Forms and Lists. When using custom delete actions, the list reference would become "Undefined" in the form data output after the initial deletion, despite the successful deletion.
+- Fixed an issue with Form outputs retrieved from Lists displaying inconsistent structure across different component hierarchies and object structures.
+- BugSnag and Mixpanel event tracking is now disabled by default during WaveMaker Enterprise (WME) setup. We re-enable event tracking later for error and usage insights (with consent).
```

Comment:
- BugSnag and Mixpanel event tracking is now disabled by default during WaveMaker Enterprise (WME) setup. We re-enable event tracking later for error and usage insights (with consent).
i think its not fixed yet. Talk to Developer